src/shogun/machine/GLM.cpp

```diff
@@ -0,0 +1,130 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Author: Tej Sukhatme
+ */
+
+#include <shogun/features/DotFeatures.h>
+#include <shogun/labels/Labels.h>
+#include <shogun/lib/observers/ObservedValueTemplated.h>
+#include <shogun/labels/RegressionLabels.h>
+#include <shogun/machine/LinearMachine.h>
+#include <shogun/machine/GLM.h>
+#include <shogun/mathematics/linalg/LinalgNamespace.h>
+#include <shogun/mathematics/NormalDistribution.h>
+#include <shogun/mathematics/RandomNamespace.h>
+#include <shogun/optimization/ElasticNetPenalty.h>
+#include <shogun/optimization/SGDMinimizer.h>
+#include <shogun/optimization/GradientDescendUpdater.h>
+#include <shogun/optimization/ConstLearningRate.h>
+
+#include<iostream>
+// #include <utility>
+#include <cmath>
+
+using namespace shogun;
+
+
+
+
+
+GLM::GLM()
+{
+	std::cout<<"Entered non-parameterized constructor.\n";
```

Comment:
Please remove all of these statements. Usually, when you push code on the PR, it should be free of these `std::cout` methods. If you want to keep them to make it easier for you to check things, you can replace them with calls to `SG_DEBUG` which are allowed:

```c++
SG_DEBUG("Entering GLM constructor.")
```

Reply:
Okay.

Reply:
How do you get SG_DEBUG() messages to be displayed on the console when I run the test?

Reply:
you need to compile in debug mode, and then it should appear automatically, because default logging level should be debug